src/binary/decimal.rs

```diff
@@ -3,14 +3,26 @@
 use std::io::Write;
 
 use arrayvec::ArrayVec;
+use bigdecimal::Zero;
 
 use crate::{
     binary::{int::Int, var_int::VarInt, var_uint::VarUInt, writer::MAX_INLINE_LENGTH},
     result::IonResult,
-    types::{coefficient::Sign, decimal::Decimal, magnitude::Magnitude},
+    types::{
+        coefficient::{Coefficient, Sign},
+        decimal::Decimal,
+        magnitude::Magnitude,
+    },
 };
 
 const DECIMAL_BUFFER_SIZE: usize = 16;
+const DECIMAL_POSITIVE_ZERO: Decimal = Decimal {
+    coefficient: Coefficient {
+        sign: Sign::Positive,
+        magnitude: Magnitude::U64(0),
+    },
+    exponent: 0,
+};
```

Comment:
This is a specific positive zero--is this constant modeling for that specific case?

Reply:
Responded in other comment.